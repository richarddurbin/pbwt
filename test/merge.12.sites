20	60479	C	T
20	60522	T	C
20	60571	C	A
20	60795	G	C
20	60828	T	G
20	61182	C	T
20	61270	A	C
20	61279	C	T
20	61329	C	T
