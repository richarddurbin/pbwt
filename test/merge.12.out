1000010010000100
0100001001000010
0010000100100001
0001000000010000
0000100000001000
0000001000000010
1000000110000001
0100001001000010
0010010000100100
